geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
+
+NAME = AsCl3 2009
+# Point Group: C3v
+# Shape: Trigonal pyramidal
+# Bond lengths: rh1     (angstroms)
+# Bond angles:  thetah1 (degrees)
+# Source Paper: Noble-Eddy R, Masters SL, Rankin DWH, Robertson HE, Guillemin JC (2010) Molecular structures of
+vinylarsine, vinyldichloroarsine and arsine studied by gas-phase electron diffraction and quantum chemical
+calculations. J Mol Struct 978 (1-3):26-34 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+As Cl    2.164(4)
+Cl As Cl 98.9(3) 
+
+NAME = AsHO
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Grimminger R, Clouthier DJ (2011) The electronic spectrum of the previously unknown HAsO transient
+molecule. J Chem Phys 135(18):184308/1-184308/8
+doi:10.1063/1.3658645
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H As   1.576(3)
+As=O   1.6342(5)
+H As=O 101.5(4)
+
+NAME = AsH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Grimminger RA, Clouthier DJ (2012) Toward an improved understanding of the AsH2 free radical: Laser
+spectroscopy, ab initio calculations, and normal coordinate analysis. J Chem Phys 137(22):224307/1-224307/12
+http://dx.doi.org/10.1063/1.4769778
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+As H    1.519(1)
+H  As H 90.75(9)
+
+# Bond lengths: rz     (angstroms)
+# Bond angles:  thetaz (degrees)
+
+As H    1.5322(8)
+H  As H 90.7(2)
+
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+
+As H    1.5169(8)
+H  As H 90.7(2)
+
+NAME = AsP3 2010
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: rg     (angstroms)
+# Source Paper: Cossairt BM, Cummins CC, Head AR, Lichtenberger DL, Berger RJF, Hayes SA, Mitzel NW, Wu G (2010)
+On the molecular and electronic structures of AsP3 and P4. J Am Chem Soc 132 (24):8459-8465
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+As P 2.1949(28)
+P  P 2.3041(12)
+
+NAME = AuClH2
+# Point Group: C2v
+# Shape: Linear --- Linear
+# Bond lengths: reff     (angstroms)
+# Source Paper: Obenchain DA, Frank DS, Grubbs GS, Pickett HM, Novick SE (2017) The covalent interaction between
+dihydrogen and gold: A rotational spectroscopic study of H2-AuCl. J Chem Phys 146(20):204302/1-204302/7
+[http://dx.doi.org/10.1063/1.4983042]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H H 0.92(1)
+
+NAME = BFH
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Clouthier DJ (2009) The electronic spectrum of the fluoroborane free radical. II. Analysis of laserinduced fluorescence and single vibronic level emission spectra. J Chem Phys 130(16):164310/1-164310/10
+doi: 10.1063/1.3122031
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.214(2)
+B F   1.3034(5)
+F B H 120.7(1)
+
+NAME = BFH2O
+# Point Group: Cs (syn)
+# Shape: Planar
+# Bond lengths: re(se)     (angstroms)
```

Comment:
Maybe a superscript for "se" is better than parentheses, for the same reasons why I said earlier that this was the case for r^M vs r(M).

Reply:
Fixed.
```
# Bond lengths: re^se     (angstroms)
# Bond angles:  thetae^se (degrees)
```